lib/philomena_web/templates/layout/_footer.html.slime

```diff
@@ -5,13 +5,20 @@ footer#footer
     = for column <- footer_data["cols"] do
       .footercol
         h5 = column
-        
+
         = for row <- footer_data[column] do
           = if row["bold"] do
-            strong
-              a href=row["url"] = row["title"]
+            = if row["new_tab"] do
```

Comment:
This is a little bit unclean due to the nesting. Can we use
```elixir
target = row["target"] || "_self"
```
and then
```slim
a href=row["url"] target=target = row["title"]
```
instead?

Reply:
Why not doing it on the same line? Check last commit.